instructure/canvas-android
# [Parent][MBL-14237] Course routing for syllabus/frontpage

To Test:
Click links (internally/externally) to either the course syllabus, or the course front page. This should open the new course detail view with just that specific element in it.
